versioned/spi/src/main/java/org/projectnessie/versioned/VersionStore.java

```diff
@@ -382,20 +382,33 @@ List<IdentifiedContentKey> getIdentifiedKeys(Ref ref, Collection<ContentKey> key
    *
    * @param ref Any ref type allowed
    * @param key The key for the specific value
-   * @return The value.
+   * @param returnNotFound whether to return a non-{@code null} result object with a {@code null}
+   *     value in {@link ContentResult#content()} instead of a {@code null} return-value for
+   *     non-existing content
+   * @return The value or {@code null} if the key does not exist and {@code returnNotFound} is
+   *     {@code false}. If the content does not exist and {@code returnNotFound} is {@code true},
+   *     {@link ContentResult#content()} will be {@code null}.
    * @throws ReferenceNotFoundException if {@code ref} is not present in the store
    */
-  ContentResult getValue(Ref ref, ContentKey key) throws ReferenceNotFoundException;
+  ContentResult getValue(Ref ref, ContentKey key, boolean returnNotFound)
```

Comment:
Shouldn't we either:

1. keep the old methods as delegates to the new ones? Could also minimize the impact since existing callers wouldn't need to be aware of this change.
2. Or, conversely, generalize the new behavior and adapt existing callers? (Might be a bit dangerous though, since it would be a behavioral change without any binary change).

Reply:
Nah - it's a "boring" change, and we'd have to carry the "older" one (1).